### DISCUSSION TITLE
Welcome to discord-bot-dashboard-next Discussions!

### DISCUSSION THREAD

**Discussion by fuma-nama** [SonMooSans/discord-bot-dashboard-next#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by oscarxdev** [comment]
Hey!!
I did everything as you say or so think, everything works great but when try to load the page `http://localhost:3000/guilds/` with any guild it says `Failed To Load` do you know why it could be?

**Reply by fuma-nama** [reply]
It usually caused by failed to load guild info or the backend server wasn't running.
Please check if anything went wrong at the backend side.
